src/components/TopBar/README.md

```diff
@@ -224,10 +230,30 @@ function TopBarExample() {
     />
   );
 
+  const secondaryMenuMarkup = (
+    <TopBar.Menu
+      activatorContent={
+        <span>
+          <Icon source={QuestionMarkMajorTwotone} color="white" />
+          <VisuallyHidden>Secondary menu</VisuallyHidden>
```

Comment:
Kinda curious why you went with visually hidden over the accessibility label on Icon.

Reply:
I actually prefer the accessibility label but for some reason that was still causing the accessibility tests to fail.